dev/integration_tests/android_engine_test/test_driver/hcpp/platform_view_main_test.dart

```diff
@@ -63,4 +63,16 @@ void main() async {
       matchesGoldenFile('$goldenPrefix.platform_view_portait_rotated_back.png'),
     );
   }, timeout: Timeout.none);
+
+  // Note: this doesn't reset the app so if additional test cases are added
+  // make sure to press the button again.
+  test('should remove overlay when platform view is removed', () async {
+    await flutterDriver.tap(find.byValueKey('RemoveOverlay'));
```

Comment:
What is pressing `AddOverlay` to begin with? Or is that the default state?

Reply:
Add overlay is the default state.